commit cd94aad0c8a418b76412da872dcd2abb5def4dfc
Author: Ufuk Celebi <uce@apache.org>
Date:   Wed Aug 10 12:18:12 2016 +0200

    [hotfix] [runtime] Remove unneeded check in KvStateClientTest
    
    The removed condition check was wrong and lead to unnecessarily long
    wait times before the test could succeed.

diff --git a/flink-runtime/src/test/java/org/apache/flink/runtime/query/netty/KvStateClientTest.java b/flink-runtime/src/test/java/org/apache/flink/runtime/query/netty/KvStateClientTest.java
index 72d9f61..ac03f94 100644
--- a/flink-runtime/src/test/java/org/apache/flink/runtime/query/netty/KvStateClientTest.java
+++ b/flink-runtime/src/test/java/org/apache/flink/runtime/query/netty/KvStateClientTest.java
@@ -318,8 +318,7 @@ public class KvStateClientTest {
 			int totalQueries = numQueryTasks * numQueriesPerTask;
 
 			// Counts can take some time to propagate
-			while (deadline.hasTimeLeft() && (stats.getNumSuccessful() != totalQueries ||
-					stats.getNumFailed() != totalQueries)) {
+			while (deadline.hasTimeLeft() && stats.getNumSuccessful() != totalQueries) {
 				Thread.sleep(100);
 			}
 
